configure.ac

```
@@ -462,7 +462,7 @@ AC_ARG_WITH(systemddir,
             [systemddir=$withval],
             [systemddir='${prefix}/lib/systemd/system'])
 AC_SUBST(systemddir)
-AM_CONDITIONAL([USE_SYSTEMD], test [ -d '/usr/lib/systemd/system' ])
+AM_CONDITIONAL([USE_SYSTEMD], test [ -d '/usr/lib/systemd/system/' -o -d '/usr/lib/systemd/'])
```

If first condition is true, second one will also be true. If first one is false, the result depends only on the second one.

Shouldn't be better to only check for `/usr/lib/systemd` ?

```suggestion
AM_CONDITIONAL([USE_SYSTEMD], test [ -d '/usr/lib/systemd/' ])
```

---

better? For some definition of better I guess. I answered this in the commit message.